src/iuse.cpp

```diff
@@ -7107,27 +7112,27 @@ static extended_photo_def photo_def_for_camera_point( const tripoint aim_point,
 
     if( !obj_coll.items.empty() ) {
         std::string obj_list = enumerate_as_string( obj_coll.items.begin(), obj_coll.items.end(),
-                               format_object_pair );
-        photo_text += "\n\n" + string_format( _( "There is something lying on the ground: %s." ),
-                                              obj_list );
+                               format_object_pair_article );
+        photo_text += "\n\n" + string_format( _( "There %s something lying on the ground: %s." ),
+                                              obj_coll.items.size() == 1 ? "is" : "are", obj_list );
```

Comment:
These will be really hard to translate correctly for languages that don't have articles.

Reply:
Point taken, though I'm not quite sure what we can do about it.  Any suggestions?

Also just noticed that the grammar with "are" is off in this case, and I totally forgot to mark the linking verbs for translation in my suggestions.

I'm thinking it would be better to do something like this because it would probably be easier to translate:
```cpp
photo_text += "\n\n" + string_format( _( "%s: %s" ),
    obj_coll.items.size() == 1 ?
        _( "There is something lying on the ground" ) :
        _( "There are some things lying on the ground" ),
    obj_list );
```

Reply:
> I'm thinking it would be better to do something like this because it would probably be easier to translate:

Yeah, something like that indeed would be better.